recipes/leftbar/callbox/callbox.test.js

```diff
@@ -1,107 +1,224 @@
-    mockProvide = {};
   });
 
   describe('Presentation Tests', () => {
-    /*
-     * Test(s) to ensure that the component is correctly rendering
-     */
-    it('Should render the component', () => {
-      expect(wrapper).toBeDefined();
+    describe('When the callbox renders', () => {
+      it('Should exist', () => {
+        expect(wrapper.exists()).toBeTruthy();
+      });
     });
-  });
 
-  describe('Accessibility Tests', () => {
-    /*
-     * Test(s) to ensure that the component is accessible
-     */
-    it('Bypass tests', () => {
-      expect(true).toBeTruthy();
+    describe('When props are provided', () => {
+      describe('When badgeText is provided', () => {
+        it('Should display badge text', () => {
+          expect(badge.text()).toBe(MOCK_BADGE_TEXT);
+        });
+      });
+
+      describe('When badgeColor is provided', () => {
+        it('Should display correct badge color', () => {
+          expect(badge.classes(CALLBOX_BADGE_COLORS[MOCK_BADGE_COLOR])).toBe(true);
+        });
+      });
+
+      describe('When avatarSrc and avatarFullName are provided', () => {
+        it('Should render the image', () => {
+          mockProps = {
+            avatarSrc: MOCK_AVATAR_IMAGE_SOURCE,
+            avatarFullName: MOCK_AVATAR_FULL_NAME,
+          };
+
+          updateWrapper();
+
+          expect(avatar.exists()).toBeTruthy();
+          expect(avatarImage.attributes('src')).toBe(MOCK_AVATAR_IMAGE_SOURCE);
+        });
+      });
+
+      describe('When avatarFullName is provided', () => {
+        it('Should render the avatar initials', () => {
+          mockProps = { avatarFullName: MOCK_AVATAR_FULL_NAME };
+
+          updateWrapper();
+
+          expect(avatar.exists()).toBeTruthy();
+          expect(avatar.text()).toBe(MOCK_AVATAR_INITIALS);
+        });
+      });
+
+      describe('When title is provided', () => {
+        it('Should display title text', () => {
+          expect(title.text()).toBe(MOCK_TITLE);
+        });
+      });
+
+      describe('When borderColor is provided', () => {
+        it('Should display correct border color', () => {
+          expect(mainContent.classes(CALLBOX_BORDER_COLORS[MOCK_BORDER_COLOR])).toBe(true);
+        });
+      });
     });
-  });
 
-  describe('Interactivity Tests', () => {
-    /*
-     * Test(s) to ensure that the component correctly handles user input
-     */
-    it('Bypass tests', () => {
-      expect(true).toBeTruthy();
+    describe('When props are not provided', () => {
+      describe('When neither avatarSrc or avatarFullName are provided', () => {
+        it('Should not display the avatar', () => {
+          expect(avatar.exists()).toBeFalsy();
+        });
+      });
+    });
+
+    describe('When slots are provided', () => {
+      describe('When badge slot is provided', () => {
+        it('Renders badge slot content', () => {
+          mockSlots = { badge: MOCK_BADGE_SLOT_CONTENT };
+          updateWrapper();
```

Comment:
break line between `mockSlots` and `updateWrapper();`

Reply:
Fixed, thanks!